tools/pepsi/src/aliveness.rs

```diff
@@ -26,15 +26,25 @@ pub async fn aliveness(arguments: Arguments) -> anyhow::Result<()> {
         Arguments::Disable { naos } => (false, naos),
     };
 
-    let tasks = naos.into_iter().map(|nao_address| async move {
-        let nao = Nao::new(nao_address.ip);
-        nao.set_aliveness(enable)
-            .await
-            .with_context(|| format!("Failed to set aliveness on {nao_address}"))
+    let multi_progress = ProgressIndicator::new();
+
+    let tasks = naos.into_iter().map(|nao_address| {
+        let multi_progress = multi_progress.clone();
+        async move {
+            let progress = multi_progress.task(nao_address.to_string());
+            progress.set_message("Setting aliveness...");
+
+            let nao = Nao::new(nao_address.ip);
+
+            progress.finish_with(
+                nao.set_aliveness(enable)
+                    .await
+                    .with_context(|| format!("Failed to set aliveness on {nao_address}")),
```

Comment:
We discussed in some Dev-Meeting that we want to transition to lowercase error messages by default. Do the case-style need to match the status messages? Maybe also write them lowercase?

Reply:
This was fixed after rebasing on main.